src/main/com/fulcrologic/fulcro/server/api_middleware.clj

```diff
@@ -18,12 +18,10 @@
      :body    "Invalid request."}))
 
 (defn generate-response
-  "Generate a Fulcro-compatible response containing at least a status code, headers, and body. You should
-  pre-populate at least the body of the input-response.
-  The content type of the returned response will always be pegged to 'application/transit+json'."
-  [{:keys [status body headers] :or {status 200} :as input-response}]
-  (-> (assoc input-response :status status :body body)
-    (update :headers assoc "Content-Type" "application/transit+json")))
+  "Generate a Fulcro-compatible response containing at least a status code, and
+  body. You should pre-populate at least the body of the input-response."
+  [{:keys [status body] :or {status 200} :as input-response}]
+  (assoc input-response :status status :body body))
```

Comment:
This is OK because of https://github.com/fulcrologic/fulcro/blob/develop/src/main/com/fulcrologic/fulcro/server/api_middleware.clj#L174